src/components/Subpage.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import { Card, CardProps } from "antd";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
+import { Diagram } from "../types";
+
+const {
+    container,
+    card,
+    dataTable,
+    dataRow,
+    dataLabel,
+    diagramCard,
+    diagramCardHeader,
+    diagramCardFooter,
+    diagramImage,
+    legendText,
+} = require("../style/disease-subpage.module.css");
+
+export interface DataRow {
+    label: string;
+    value: string;
+    formatClass?: string;
+}
+
+export interface SubpageProps {
+    data: {
+        rows: DataRow[];
+        diagrams?: Diagram[];
+        legendContent?: React.ReactNode;
+    };
+    headerLeadText?: string;
+}
+
+interface DiagramCardProps extends CardProps {
+    title?: string;
+    footer?: React.ReactNode;
+}
+
+export const DiagramCard: React.FC<DiagramCardProps> = ({
```

Comment:
I'm not sure if subpage makes sense as a separate component, but this DiagramCard should definitely be it's own component, because this will be reused a lot